ArduPilot/ardupilot
# AP_Logger: Prepare the maximum number of log files in the config para…

…meter

rework of https://github.com/ArduPilot/ardupilot/pull/10146

Tested on SITL, Same behavior as master : 
- make a fly, and rtl to have log
- log list to check the log is there and new one is created on next takeoff
- create 140 logs (copy previous log and rename, set LASTLOG.txt to 140
- log list to see 140 logs entries
- set LOG_MAX_FILES to 140
- reboot
- log list to see 140 log
- fly, rtl, wait disarm: still 140 logs


Gain from this : 
- faster time to gather the log list as less to look for, when you got automated flight, those second are importants
